jcache/embedded/src/main/java/org/infinispan/jcache/annotation/InjectedCacheResolver.java

```diff
@@ -97,13 +120,34 @@ private JCacheManager toJCacheManager(EmbeddedCacheManager cacheManager) {
       return getCacheFromDefaultCacheManager(cacheName);
    }
 
-   private <K, V> Cache<K, V> getCacheFromDefaultCacheManager(String cacheName) {
-      Cache<K, V> cache = defaultJCacheManager.getCache(cacheName);
-      if (cache != null)
-         return cache;
+   private <K, V> Cache<K, V> getCacheFromDefaultCacheManager(final String cacheName) {
+      final Configuration defaultInjectedConfiguration = getBeanReference(Configuration.class);
```

Comment:
yep in cdi tests we override default producer and use custom one wich produces configurations for caches